std/assembly/number.ts

```diff
@@ -331,23 +345,23 @@ export abstract class F64 {
   static readonly NaN: f64 = NaN;
 
   static isNaN(value: f64): bool {
-    return builtin_isNaN<f64>(value);
+    return isNaN<f64>(value);
   }
 
   static isFinite(value: f64): bool {
-    return builtin_isFinite<f64>(value);
+    return isFinite<f64>(value);
```

Comment:
Same. Not sure how TS handles this if general signatures differ, like one being generic and the other not.